internals/overlord/overlord.go

```diff
@@ -136,18 +138,27 @@ func New(opts *Options) (*Overlord, error) {
 	}
 	o.runner.AddOptionalHandler(matchAnyUnknownTask, handleUnknownTask, nil)
 
+	o.planMgr, err = planstate.NewManager(s, o.runner, o.pebbleDir)
+	if err != nil {
+		return nil, err
```

Comment:
No need to wrap the error here?

Reply:
Yes, @thp-canonical your comment made sense all along. I recently reviewed another PR and realised that "common" context makes sense at the consumer side, not inside the function, otherwise you would have tons of error suffixes all starting with "cannot create plan manager" ...

Reply:
Fixed!